Source/ProgressionSystemRuntime/Private/PSCheatExtension.cpp

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Valerii Rotermel & Yevhenii Selivanov
+
+
+#include "PSCheatExtension.h"
+
+#include "Components/PSHUDComponent.h"
+#include "Data/PSWorldSubsystem.h"
+
```

Comment:
You forgot to add `UE_INLINE_GENERATED_CPP_BY_NAME`. Please be careful to what Rider tells you, your plugin shouldn't populate any warnings, you can see them in the 'Build' window

Reply:
done https://github.com/h4rdmol/ProgressionSystem/pull/1/commits/077a0b0aaaf6efacad188459059baece40744a23

Reply:
Please be careful (!!) to warnings Rider shows you. I still see there are some of them, like `UE_INLINE_GENERATED_CPP_BY_NAME` missing in the `PSStarWidget`

Reply:
my bad. not always easy to see them in tons of logs. Sovled https://github.com/h4rdmol/ProgressionSystem/pull/1/commits/de5b19b60fed410181a4df481d47c3cdde17dde6